exercises/concept/temperature/.docs/instructions.md

```diff
@@ -0,0 +1,17 @@
+# Instructions
+
+## What's the Temperature?
+
+Write a function `tempToC` to convert an integer temperature in Fahrenheit to its equivalent in Celsius rounded to the nearest hundredth. It should work like this:
+
+```haskell
+> tempToC 32
+0.00
+```
+
+Write a second function, `tempToF` to convert a decimal temperature in Celsius to its equivalent in Fahrenheit. For the safety of your American team mate, round the result to the **next** integer. It should work like this:
+
+```Haskell
+> tempToF 0.00
+32
+```
```

Comment:
The format used here is slightly different from what we require.
The key thing is that there are no individual tasks being defined (they are grouped together). 
See https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-instructions-md for more information on how to structure the `instructions.md` document.